src/components/pages/AdminHome/AdminSidebar.js

```diff
@@ -68,11 +68,11 @@ function AdminSidebar() {
           <a href="/admin-purchases">Purchases</a>
         </Menu.Item>
 
-        <Menu.Item key="6">
+        <Menu.Item key="8">
```

Comment:
I'm not familiar with the codebase but I generally steer clear of manually setting keys. My recommendation would be to create an array of objects with everything needed to build out a menu item and loop through the array to build these. That way the index will always auto-adjust and you just have to change the array / object to make changes